xyz.stuerz.BilligSweeper.metainfo.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Benjamin Stürz -->
+<component type="desktop-application">
```

Comment:
this file should be managed upstream in a future release.

Reply:
Ok, just copy the file from the project?
Or upload it to some database?

Reply:
it it is upstream it has no reason to be here.

Reply:
Ok, I'm gonna note it for the next release.